spark-connector/src/main/java/com/datastrato/gravitino/spark/catalog/GravitinoCatalog.java

```diff
@@ -175,7 +178,29 @@ public void createNamespace(String[] namespace, Map<String, String> metadata)
   @Override
   public void alterNamespace(String[] namespace, NamespaceChange... changes)
       throws NoSuchNamespaceException {
-    throw new NotSupportedException("Doesn't support altering namespace");
+    validateNamespace(namespace);
+    SchemaChange[] schemaChanges =
+        Arrays.stream(changes)
+            .map(
+                change -> {
+                  if (change instanceof SetProperty) {
+                    SetProperty setProperty = ((SetProperty) change);
+                    return SchemaChange.setProperty(setProperty.property(), setProperty.value());
+                  } else if (change instanceof RemoveProperty) {
+                    RemoveProperty removeProperty = ((RemoveProperty) change);
+                    return SchemaChange.removeProperty(removeProperty.property());
+                  } else {
+                    throw new UnsupportedOperationException("Unsupported namespace change");
```

Comment:
Could you add detailed Unsupported `SchemaChange` to the error message in case SparkSQL add a new `SchemaChange`?